applications/portfolioDrafts/getPortfolioDrafts.ts

```diff
@@ -0,0 +1,35 @@
+import { ScanCommand, ScanCommandInput } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyResult } from "aws-lambda";
+import { ErrorCodes } from "./models/Error";
+import { PortfolioFull } from "./models/PortfolioFull";
+import { dynamodbDocumentClient as client } from "./utils/dynamodb";
+import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "./utils/response";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+
+/**
+ * Gets all Portfolio Drafts, TODO: "...to which the user has read access"
+ * Revisit once authentication and authorization are in place.
+ * @param event - The GET request from API Gateway
+ */
+export const handler = async (): Promise<APIGatewayProxyResult> => {
+  const params: ScanCommandInput = {
+    TableName: TABLE_NAME,
+  };
+
+  // This is an expensive command and should be replaced when
+  // portfolio draft owners and authenticated users are available
+  // so that "...to which the user has read access" can be added to the implementation.
+  const command = new ScanCommand(params);
+
+  try {
+    const data = await client.send(command);
+    return new ApiSuccessResponse<PortfolioFull[]>(data.Items as PortfolioFull[], SuccessStatusCode.OK);
```

Comment:
I think per the API spec, this should be a `PortfolioDraftSummary`. It's possible that you can either do this by only casting to a `PortfolioDraft` (but that's probably not good enough). You might need a `ProjectionExpression` argument as part of `ScanCommandInput` that only selects the attributes of a `PortfolioDraft`

Reply:
ProjectionExpression docs: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-dynamodb/interfaces/scancommandinput.html#projectionexpression

Reply:
Had been using PortfolioFull because UI needs a count of the portfolio managers, however, recent API changes will allow an improved approach by including such a count in the summary.

Of course we need a solution to update that, but it's another matter.  Maybe a separate function triggered by `createPortfolioStep`.

Reply:
Models renamed in #59

Reply:
API spec and the function impl has been changed to return an array of PortfolioDraftSummary.